Source/ACE.Server/WorldObjects/WorldObject_Properties.cs

```diff
@@ -2409,12 +2409,13 @@ public string QuestRestriction
         /* pressure plates */
 
         /// <summary>
-        /// Returns TRUE if this object can be activated (default)
+        /// Returns 1 or 65535 if this object can be activated (1 is default)
+        /// 23 Weenies in Cache.bin contain Active 65535
```

Comment:
should be 0xFFFF

Reply:
I was going to change it to a descriptive Enum